backend/accounts_test.go

```diff
@@ -179,3 +182,86 @@ func TestNextAccountNumber(t *testing.T) {
 	_, err = nextAccountNumber(coinpkg.CodeTBTC, ks(fingerprint2, true), accountsConfig)
 	require.Equal(t, ErrAccountLimitReached, errp.Cause(err))
 }
+
+func newBackend(t *testing.T, testing, regtest bool) *Backend {
```

Comment:
Consider defining consts for the booleans:

```go
const (
  testnetEnabled = true
  testnetDisabled = false
  regtestEnabled = true
  regtestDisabled = false
)
```

It just makes reading anywhere else easier:

```go
newBackend(t, testnetEnabled, regtestDisabled)
NewArguments(.., testnetDisabled, regtestDisabled, ...)
```

Also, `testing` sounds like it's running some tests? It's testnet, no?

Reply:
Done.


> Also, testing sounds like it's running some tests? It's testnet, no?

What do you mean? It enables testnet mode, which means that tbtc, tltc, teth, reth are available and the mainnet coins are not.